cabal.project

```diff
@@ -40,12 +42,6 @@ constraints:
   -- in the future, thus: TODO: remove this flag.
   bitvec -simd,
 
-
-
 if impl(ghc >= 9.9)
-  allow-newer:
-    haddock-library:base,
-    haddock-library:containers,
+  -- https://github.com/haskell/haskell-language-server/issues/4324
   benchmarks: False
-else
-  benchmarks: True
```

Comment:
I guess `benchmarks: True` is default right?

Reply:
I moved it up above.